lxd/lxd-metadata/lxd_metadata.go

```diff
@@ -432,41 +432,37 @@ func writeDocFile(inputJSONPath, outputTxtPath string) error {
 							quoteFormattedValue = fmt.Sprintf("\"%s\"", configContentValueStr)
 						}
 
-						kvBuffer.WriteString(
-							fmt.Sprintf(
-								":%s: %s\n",
-								configEntryContentKey,
-								quoteFormattedValue,
-							),
-						)
+						fmt.Fprintf(kvBuffer,
+
```

Comment:
is this empty line needed?

Reply:
Nope not needed nor wanted but seems to be an artifact from the automatic tool used to fix it. It seems that it was confused by the `"` being on a different line than the opening parenthesis of `Sprintf(`.